app/models/auxiliar_business_unit_in_plan_item.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+class AuxiliarBusinessUnitInPlanItem < ApplicationRecord
```

Comment:
Le cambiaría el nombre por algo un toque más corto, aunque lo usemos _poco_:
```suggestion
class AuxiliarBusinessUnit < ApplicationRecord
```

Reply:
dale perfecto